apple/internal/partials/swift_dylibs.bzl

```diff
@@ -191,6 +202,7 @@ def _swift_dylibs_partial_impl(
                 platform_prerequisites = platform_prerequisites,
                 resolved_swift_stdlib_tool = apple_toolchain_info.resolved_swift_stdlib_tool,
                 strip_bitcode = strip_bitcode,
+                strip_swift_symbols = strip_swift_symbols,
```

Comment:
JFYI, I tested this on our TestFlight builds.
Enabling `strip_swift_symbols` in the `swift_dylibs` made the TestFlight validation fail, but enabling it just in `framework_import` was OK.

```
ITMS-90428: Invalid Swift Support - The files libswiftos.dylib, libswiftARKit.dylib, ... (long list)
```

Reply:
What was the end of the error message?

Reply:
Oops sorry.

```
ITMS-90428: Invalid Swift Support - The files libswiftos.dylib, (long list), don’t match /Payload/(app name).app/Frameworks/libswiftos.dylib, (long list).
```